docs/en_US/jailbreak/installing-palera1n-tvos-15.0.md

```diff
@@ -0,0 +1,250 @@
+
+### Additional Requirements
+
+- *These are in addition to the ones at the top of this guide*
+- A 128MB or greater USB Drive
+    - If you don't have a USB drive but have another form of supported removable bootable hardware (e.g., an SD card or DVD), you can use that instead.
+- The latest version of [palen1x](https://github.com/palera1n/palen1x/releases)
+- [Ventoy](https://github.com/ventoy/Ventoy/releases)
+
+
+### Installing Ventoy
+
+1. Download and extract the contents of the `Ventoy.zip` file.
+1. Insert your USB drive if you have not already done so, and open the `Ventoy2Disk.exe` file.
+1. Select the USB drive you would like to boot palen1x from.
+    - This USB drive will be completely erased.
+    - Ensure you back up all important data beforehand.
+1. Click `Install` and confirm that you are okay with erasing the USB drive.
+    - Do not remove your USB drive until the process has completed.
+1. Once it has installed, copy the palen1x `.iso` file that you downloaded onto the USB drive.
+
+### Booting palen1x
+
+1. Reboot your computer, go into your BIOS settings to disable Secure Boot, then enter the boot picker and select your USB drive to boot from.
+    - This process is different for every computer.
+    - Search for your PC or motherboard brand to find out how to get to your BIOS settings if you are unsure.
+1. Press `Enter` once you see the Ventoy screen to boot into palen1x.
+
+### Running palera1n
+
+::: details Apple TV 4K instructions
+1. Once you have loaded palen1x, select `Shell`.
+1. Plug your GoldenEye/Foxlink cable into the Ethernet port on your Apple TV.
+1. Plug your DCSD cable into the Lightning connector on the GoldenEye/Foxlink cable upside down-the side labeled "this side up" or similar should face down; otherwise, it might not enter <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>.
+1. Finally, plug the USB end of your DCSD cable into your computer or any power source.
+1. Restart your Apple TV with both cables plugged in to put it into DFU mode.
+1. Once in DFU mode, unplug your DCSD cable (**not** your GoldenEye/Foxlink cable, keep that plugged in) and plug in your Lightning cable, with the other end connected to your computer.
+1. Run `palera1n -f`.
+
+You'll need to do this every time you rejailbreak your device.
+:::
+
+::: details Apple TV HD instructions
+1. Once you have loaded palen1x, select `Shell`.
+1. Plug your USB-C cable into the USB-C diagnostic port on the back of the device and the other end into your computer.
+1. Run `palera1n -f`.
+1. When ready, press `Enter` and follow the on-screen instructions to enter <router-link to="/faq/#what-is-dfu-mode">DFU mode</router-link>.
+
+You'll need to do this every time you rejailbreak your device.
+:::
+
+::: tip
+
+A9(X) and earlier devices have an issue where they will get stuck midway through this process in pongoOS. To work around this, you'll need to do the following:
```

Comment:
Does this apply to Apple TV's? If it does, the start of this sentence should be changed to "The Apple TV HD has an issue".